addons/l10n_in/data/account.account.tag.csv

```diff
@@ -45,3 +45,45 @@
 "tax_tag_cess_rc","CESS (RC)","taxes"
 "tax_tag_non_itc_cess_rc","NON ITC CESS (RC)","taxes"
 "tax_tag_other_non_itc_cess_rc","Other NON ITC CESS (RC)","taxes"
+"account_tag_share_capital","Share capital","accounts"
```

Comment:
I still think that it would be better to add a country_id column here like in other l10n modules